mu/interface/panes.py

```diff
@@ -407,6 +410,34 @@ def on_get(self, microbit_file):
         self.set_message.emit(msg)
         self.list_files.emit()
 
+    def contextMenuEvent(self, event):
+        menu = QMenu(self)
+        local_filename = self.currentItem().text()
+        # Get the file extension
+        ext = os.path.splitext(local_filename)[1].lower()
```

Comment:
Will this work with files that don't have an extension (a distinct possibility when there are kids involved)..?

Reply:
Yes, splitext is actually returning a turple not list and [1] is guaranteed to be atleast an empty string